### PR TITLE
[docs][Release/2.1.x] Update to how to and command ref

### PR DESCRIPTION
**Pull request by halsaphi**
## Change Description

Update how to docs :

buy ram
update account keys
connect to a network
connect to a keosd

Update command ref docs :

cleos system buyram
cleos create key
cleos wallet import
cleos set account
cleos get account

## Change Type
**Select *ONE*:**
- [x] Documentation

- [ ] Stability bug fix

- [ ] Other

- [ ] Other - special case

## Testing Changes
**Select *ANY* that apply:**
- [ ] New Tests

- [ ] Existing Tests

- [ ] Test Framework

- [ ] CI System

- [ ] Other

## Consensus Changes
- [ ] Consensus Changes

## API Changes
- [ ] API Changes

## Documentation Additions
- [ ] Documentation Additions

### CODE REVIEW

File: docs/02_cleos/03_command-reference/create/key.md [hunk 1]
```diff
@@ -1,28 +1,62 @@
...
+### Positionals:
+- none
+### Options
+- `-h,--help` - Print this help message and exit
+- `--r1` - Generate a key using the R1 curve (iPhone), instead of the K1 curve (Bitcoin)
+`-f`,`--file` _TEXT_ - Name of file to write private/public key output to. (Must be set, unless "--to-console" is passed
+`--to-console` - Print private/public keys to console.
+
+## Requirements
+* Install the currently supported version of `cleos.`
```
Comment (lparisc) [inline docs/02_cleos/03_command-reference/create/key.md:24]: \`cleos.\` -> \`cleos\`.

File: docs/02_cleos/03_command-reference/get/account.md [hunk 2]
```diff
@@ -1,20 +1,42 @@
...
+- `core-symbol` _TEXT_ - The expected core symbol of the chain you are querying - OPTIONAL
 
-## Options
+### Options
+- `-h,--help` Print this help message and exit
 - `-j,--json` - Output in JSON format
 
+## Requirements
+* Install the currently supported version of `cleos.`
+[[info | Note]] | The cleos tool is bundled with the EOSIO software. [Installing EOSIO](../../00_install/index.md) will also install the cleos comand line tool and the keosd key store or wallet.
```
Comment (lparisc) [inline docs/02_cleos/03_command-reference/get/account.md:26]: The pipe | should start at the next line.

The cleos tool -> \`cleos\`

The phrase "or wallet" is not accurate. Consider simplifying to:
... will also install the \`cleos\` and \`keosd\` command-line tools.

File: docs/02_cleos/03_command-reference/set/set-account-permissions.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+-  `-r`,`--ref-block` _TEXT_ Set the reference block num or block id used for TAPOS (Transaction as Proof-of-Stake)
+-  `--use-old-rpc` Use old RPC push_transaction, rather than new RPC send_transaction
+-  `-p`,`--permission` _TEXT_ An account and permission level to authorize, as in 'account@permission' (defaults to 'account@active')
+-  `--max-cpu-usage-ms` _UINT_ Set an upper limit on the milliseconds of cpu usage budget, for the execution of the transaction (defaults to 0 which means no limit)
+-  `--max-net-usage` _UINT_ Set an upper limit on the net usage budget, in bytes, for the transaction (defaults to 0 which means no limit)
+- `--delay-sec` _UINT_ Set the delay_sec seconds, defaults to 0s
+
+## Requirements
+* Install the currently supported version of `cleos.`
+[[info | Note]] | The cleos tool is bundled with the EOSIO software. [Installing EOSIO](../../00_install/index.md) will also install the cleos comand line tool and the keosd key store or wallet.
```
Comment (lparisc) [inline docs/02_cleos/03_command-reference/set/set-account-permissions.md:42]: The pipe | should start at the next line.

The cleos tool -> \`cleos\`

The phrase "or wallet" is not accurate. Consider simplifying to:
... will also install the \`cleos\` and \`keosd\` command-line tools.

File: docs/02_cleos/03_command-reference/set/set-account-permissions.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+-  `--use-old-rpc` Use old RPC push_transaction, rather than new RPC send_transaction
+-  `-p`,`--permission` _TEXT_ An account and permission level to authorize, as in 'account@permission' (defaults to 'account@active')
+-  `--max-cpu-usage-ms` _UINT_ Set an upper limit on the milliseconds of cpu usage budget, for the execution of the transaction (defaults to 0 which means no limit)
+-  `--max-net-usage` _UINT_ Set an upper limit on the net usage budget, in bytes, for the transaction (defaults to 0 which means no limit)
+- `--delay-sec` _UINT_ Set the delay_sec seconds, defaults to 0s
+
+## Requirements
+* Install the currently supported version of `cleos.`
+[[info | Note]] | The cleos tool is bundled with the EOSIO software. [Installing EOSIO](../../00_install/index.md) will also install the cleos comand line tool and the keosd key store or wallet. 
+* You have access to a blockchain.
```
Comment (lparisc) [inline docs/02_cleos/03_command-reference/set/set-account-permissions.md:43]: a blockchain -> an EOSIO blockchain

File: docs/02_cleos/03_command-reference/wallet/create.md [hunk 5]
```diff
@@ -7,4 +7,85 @@ None
...
+### Options
+`-h`, `--help` - Print this help message and exit
+`-n`, `--name` _TEXT_ - The name of the new wallet, if blank the name is `default`
+`-f`, `--file` _TEXT_ - Name of file to write wallet password output to. Must be set, unless "--to-console" is passed
+`--to-console` - Print password to console.
+
+## Requirements
+* Install the currently supported version of `cleos` and `keosd`.
+[[info | Note]]
+| The `cleos` tool and `keosd` wallet is bundled with the EOSIO software. [Installing EOSIO](../../00_install/index.md) will also install the cleos comand line tool and the keosd key store or wallet.
```
Comment (lparisc) [inline docs/02_cleos/03_command-reference/wallet/create.md:37]: The cleos tool -> \`cleos\`

The phrase "or wallet" is not accurate. Consider simplifying to:
... will also install the \`cleos\` and \`keosd\` command-line tools.

File: docs/02_cleos/03_command-reference/wallet/import.md [hunk 6]
```diff
@@ -1,9 +1,55 @@
...
+- `-h,--help` - Print this help message and exit
 - `-n, --name` _TEXT_ - The name of the wallet to import key into.
-- `--private-key` _TEXT_ - Private key in WIF format to import.
\ No newline at end of file
+- `--private-key` _TEXT_ - Private key in WIF format to import.
+
+## Requirements
+* Install the currently supported version of `cleos` and `keosd`.
+[[info | Note]]
+| The `cleos` tool and `keosd` wallet is bundled with the EOSIO software. [Installing EOSIO](../../00_install/index.md) will also install the cleos comand line tool and the keosd key store or wallet.
```
Comment (lparisc) [inline docs/02_cleos/03_command-reference/wallet/import.md:25]: is bundled -> are bundled

`keosd` is not a wallet. Consider simplifying to:
The \`cleos\` tool and \`keosd\` wallet is bundled -> \`cleos\` and \`keosd\` are bundled

The phrase "or wallet" is not accurate. Consider simplifying to:
... will also install the \`cleos\` and \`keosd\` command-line tools.

File: docs/02_cleos/03_command-reference/wallet/import.md [hunk 7]
```diff
@@ -1,9 +1,55 @@
...
+[[info | Note]]
+| The `cleos` tool and `keosd` wallet is bundled with the EOSIO software. [Installing EOSIO](../../00_install/index.md) will also install the cleos comand line tool and the keosd key store or wallet. 
+
+## Examples
+1. Import a private key to the default wallet. The wallet must be `open` and `unlocked.`
+```shell
+cleos wallet import
+```
+
+The command asks for the private key
```
Comment (lparisc) [inline docs/02_cleos/03_command-reference/wallet/import.md:33]: The command asks for the private key -> The command asks for the private key. Enter it:

File: docs/02_cleos/03_command-reference/wallet/import.md [hunk 8]
```diff
@@ -1,9 +1,55 @@
...
+```shell
+5KDNWQvY2seBPVUz7MiiaEDGTwACfuXu78bwZu7w2UDM9A3u3Fs
+```
+
+**Example Output**
+```shell
+private key: imported private key for: EOS5zG7PsdtzQ9achTdRtXwHieL7yyigBFiJDRAQonqBsfKyL3XhC
+```
+
+2. Import a private key to a named wallet. The wallet must be `open` and `unlocked.`
```
Comment (lparisc) [inline docs/02_cleos/03_command-reference/wallet/import.md:44]: are backticks in \`open\` and \`unlocked\` necessary? These are not system names or identifiers. If you want to emphasize you can use **bold** or *italics*.

**Review by lparisc — APPROVED** [review]
Added a few suggestions and identified a few typos.

File: docs/02_cleos/02_how-to-guides/how-to-buy-ram.md [hunk 9]
```diff
@@ -1,33 +1,42 @@
-## Goal
 
-Acquire RAM for contract deployment, database tables, and other blockchain resources
+## Overview
+This guide provides instructions on how to buy RAM for an EOSIO blockchain account using the cleos CLI tool. RAM is a system resource and is used to store smart contract state and account information.
```
[the remark this answers is not in the document]
Comment (halsaphi) [reply docs/02_cleos/02_how-to-guides/how-to-buy-ram.md:3]: updated

File: docs/02_cleos/02_how-to-guides/how-to-buy-ram.md [hunk 10]
```diff
@@ -1,33 +1,42 @@
...
-* Install the currently supported version of `cleos`
+## Before you Begin
+Make sure you meet the following requirements: 
 
+* Install the currently supported version of `cleos.`
 [[info | Note]]
-| The cleos tool is bundled with the EOSIO software. [Installing EOSIO](../../00_install/index.md) will also install the cleos tool. 
+| The cleos tool is bundled with the EOSIO software. [Installing EOSIO](../../00_install/index.md) will also install the cleos comand line tool and the keosd key store or wallet. 
+* You have access to a blockchain and the `eosio.system` reference contract from [`eosio.contracts`](https://github.com/EOSIO/eosio.contracts) repository is deployed and used to manage system resources.
+* You have an EOSIO account and access to the account private key.
```
[the remark this answers is not in the document]
Comment (halsaphi) [reply docs/02_cleos/02_how-to-guides/how-to-buy-ram.md:14]: it's possessive so I think should be accounts'

File: docs/02_cleos/03_command-reference/set/set-account-permissions.md (hunk 4 above)
Comment (halsaphi) [reply docs/02_cleos/03_command-reference/set/set-account-permissions.md:43]: updated

**Pull request merged**